src/_gettsim/config.py

```diff
@@ -129,6 +126,10 @@
     "m_kind_berücks_zeit": float,
     "m_pfleg_berücks_zeit": float,
     "y_pflichtbeitr_ab_40": float,
+    "m_anwartschaftszeit": bool,
+    "arbeitssuchend": bool,
+    "m_durchg_alg1_bezug": int,
```

Comment:
for `m_pfleg_berücks_zeit` etc. we use the type `float`. For consistency, I would also do the same for m_durchg_alg1_bezug and soz_vers_pflicht_5j